db/structure.sql

```diff
@@ -1135,6 +1135,18 @@ CREATE SEQUENCE public.analysis_items_id_seq
 ALTER SEQUENCE public.analysis_items_id_seq OWNED BY public.analysis_items.id;
 
 
+--
```

Comment:
Since only Nomen was changed, there's no reason for the `db/structure.sql` to have been modified.